notebooks/TFoS.ipynb

```diff
@@ -13,6 +13,7 @@
     "from __future__ import print_function\n",
```

Comment:
We should also update the dcos/example or?

Reply:
It just got merged, I will update this PR accordingly

Reply:
The dcos/example should be updated when we release the new version but I don't see it correlated to this TFoS notebook.